### PR TITLE
Player No-Zone obviation

### LINKED ISSUE #637: When a player has no zones attached the volume/mute should not be shown and the lack of zones should be more obvious

### PR DESCRIPTION
**Pull request by SteveMicroNova**
### What does this change intend to accomplish?

Remove volume and mute controls when there is nothing playing on the player card and player page

Aims to close #637 by removing anything that suggests audio could be playing

What is specifically achieved is removing the volume bar and mute button from both the player card on the home page and the player page when a source has no zones, as well as making the "Add Zones" badge on the player card flash slowly when there aren't any zones attached

### Checklist

* [x] Have you tested your changes and ensured they work?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] If applicable, have you updated the CHANGELOG?
* [x] If this is a UI change, have you tested it across multiple browser platforms?
* [x] If this is a UI change, have you tested across multiple viewport sizes (ie. desktop versus mobile)?

### CODE REVIEW

**Comment by SteveMicroNova** [comment]
https://github.com/micro-nova/AmpliPi/pull/747/commits/0dec1c61558b0c73ad509edadba3d0e0195fe44b
I do not like the amount of prop drilling I had to do to make this work, and I especially do not like making a new prop for a single usecase
we need to be able to override classNames and styles in parent elements in general, this comes up a lot

**Comment by SteveMicroNova** [comment]
https://github.com/micro-nova/AmpliPi/assets/114415937/48df28e7-8e21-40e2-8371-4487aface936

See video for demonstration

**Comment by linknum23** [comment]
Mind editing the checklist and selecting whats been done?

**Comment by linknum23** [comment]
I didn't noticed the pulsing in the video until the 4th watch. I think this needs to be a little more distinctive. 
I also tested this with groove salad as a background and was unable to notice the blinking.

Would changing the background color instead of alpha help here?

**Comment by linknum23** [comment]
Also I got lamplipi back up and running should be good to test on!

File: web/src/components/Chip/Chip.scss [hunk 1]
```diff
@@ -9,3 +9,33 @@
   margin: 0.25rem;
   @include general.highlight-hover;
 }
+
+
+@keyframes shakeBackground {
+  // Do not edit the steps of this animation unless you intend to not use box-shadow
```
Comment (linknum23) [inline web/src/components/Chip/Chip.scss:15]: You may want to update this comment with a message about why we want to shake the background. Something like "Get the users attention but don't be obnoxious about it. Used For things that aren't errors, but need some user attention like a source with no zones "

**Review by linknum23 — APPROVED** [review]
I like how simple this is! Nice work.

**Pull request merged**